packages/arb-token-bridge-ui/src/components/common/Dialog.tsx

```diff
@@ -3,6 +3,7 @@ import { Fragment, useCallback, useRef, useState } from 'react'
 
 import { Button, ButtonProps } from './Button'
 import { getTransitionProps } from './Transition'
+import CloseIcon from 'src/assets/close.svg'
```

Comment:
We have a package called `@heroicons/react` for these common icons, you could do:

```
import { XIcon } from '@heroicons/react/outline'
```

We do it in `WithdrawalConfirmationDialog`, too.